sync/class.jetpack-sync-actions.php

```diff
@@ -81,9 +81,21 @@ static function add_sender_shutdown() {
 				defined( 'PHPUNIT_JETPACK_TESTSUITE' )
 			)
 		) ) {
-			self::initialize_sender();
-			add_action( 'shutdown', array( self::$sender, 'do_sync' ) );
-			add_action( 'shutdown', array( self::$sender, 'do_full_sync' ) );
+			if ( isset( $_GET['jetpack_sync_async_sender'] ) ) {
+				if ( ! defined( 'JETPACK_SYNC_DOING_ASYNC_SENDER' ) ) {
+					define( 'JETPACK_SYNC_DOING_ASYNC_SENDER', true );
+				}
+				ignore_user_abort(true );
+			}
+
+			if ( ! defined( 'JETPACK_SYNC_DOING_ASYNC_SENDER' )
+			     && apply_filters( 'jetpack_sync_async_sender', defined( 'REST_REQUEST' ) && REST_REQUEST && ! headers_sent() ) ) {
```

Comment:
not sure how it could affect server load to do a parallel request on every `POST` or `is_admin ` request